data/json/items/armor.json

```diff
@@ -434,6 +454,29 @@
     "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25, "encumbrance": 1.5, "warmth": 1.5 },
     "relative": { "bashing": 1, "material_thickness": 1, "environmental_protection": 2 }
   },
+  {
+    "id": "armor_EOD_light",
+    "type": "ARMOR",
+    "name": "tactical EOD suit",
+    "description": "This is a bulky kevlar suit for explosive ordnance disposal in hostile environments. Lighter than a normal EOD suit, it compromises between protection and maneuverability, though it will still provide some protection from overpressure, fragmentation, impact and heat.",
```

Comment:
```suggestion
    "description": "This is a bulky kevlar suit for explosive ordnance disposal in hostile environments.  Lighter than a normal EOD suit, it compromises between protection and maneuverability, though it will still provide some protection from overpressure, fragmentation, impact and heat.",
```